packages/react-core/src/components/TreeView/__tests__/TreeView.test.tsx

```diff
@@ -92,7 +93,11 @@ const flagOptions = [
     name: 'Cost Management',
     id: 'Cost',
     hasBadge: true,
-    action: <FolderIcon />,
+    action: (
+      <Button>
```

Comment:
Can you make this a plain button and add an `aria-label`?

```suggestion
      <Button variant="plain" aria-label="Action">
```

Reply:
And update this in any other examples of a tree view action? By default `<Button>` renders as a primary variant, which we don't want to use here.

Reply:
Got it, will do.